definitions/json/CaseEvent/CaseEvent.json

```
@@ -584,8 +584,8 @@
   {
     "CaseTypeID": "ET_Scotland",
     "ID": "pseRespondentViewNotification",
-    "Name": "Judgement, Order, Notification",
-    "Description": "View a judgment, order or notification",
+    "Name": "Judgment, Order, Notification",
+    "Description": "View all judgment, orders and notifications",
```

This one shouldn't change. The one you're thinking of is https://github.com/hmcts/et-ccd-definitions-scotland/blob/4388c5a8ae08da5fc850bd4c37e13f196598804e/definitions/json/CaseEvent/CaseEvent.json#L588 which is already correct but could probably become judgments
```suggestion
    "Description": "View a judgment, order and notification",
```